### PR TITLE
Set accept-language header

### PR DESCRIPTION
**Pull request by lindskogen**

### CODE REVIEW

File: plugins/titlegiver/titlegiver.py [hunk 1]
```diff
@@ -22,7 +22,7 @@ def __init__(self):
     @staticmethod
     def get_title_from_url(url):
         # Fetch page (no need to verfiy SSL certs for titles)
-        response = auto_requests.get(url, verify=False, headers={"User-Agent": Titlegiver.USER_AGENT})
+        response = auto_requests.get(url, verify=False, headers={"User-Agent": Titlegiver.USER_AGENT, "Accept-Language": "en_US"})
```
Comment (reggna) [inline plugins/titlegiver/titlegiver.py]: Did you check to make sure that this fixes the "op"-title on Twitter?

Comment (reggna) [inline plugins/titlegiver/titlegiver.py]: Is there any point of making this configurable from the settings.json, or taking the default system lang or something like that?  I'm not against the PR in its current form, I'm just asking.

Comment (reggna) [inline plugins/titlegiver/titlegiver.py]: Also, why isn't these "sensible default" headers just part of the auto_requests library?  I mean both the user agent and the lang.

Comment (lindskogen) [inline plugins/titlegiver/titlegiver.py]: Ok, verified now that it actually fixes 'on Twitter', not sure about the configurability? @Tigge: opinions?

**Comment by Tigge** [comment]
Let's do config stuff later!

**Pull request merged**